tycho-plugins/target-platform-utils/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,61 @@
+eclipse.preferences.version=1
```

Comment:
did you intend to commit your .settings stuff? Is it required to be shared?

Reply:
Yes, it shares the formatter and save actions to keep code clean and contributor and reviewers happy.

Reply:
In that case +10!